_layouts/default.html

```diff
@@ -1,39 +1,33 @@
 <!DOCTYPE html>
 <html lang="en">
-{% include _head.html %}
+  {% include _head.html %}
 
-<body class="crds-styles">
-  {% include _gtm-body.html %}
-  {% if page.requires_auth %}
-  {% include _preloader.html %}
-  {% endif %}
-  {{ content }}
-  {% include _monetate-body.html %}
-  <script>
-    var CRDS = {
-      media: {
-        cms: '{{ site.shared_header.cms }}',
-        app: '{{ site.shared_header.app }}',
-        img: '{{ site.shared_header.img }}',
-        prefix: '{{ site.shared_header.prefix }}'
-      },
-      streamspotId: '{{ site.streamspotId }}',
-      streamspotKey: '{{ site.streamspotKey }}',
-      streamspotPlayerId: '{{ site.streamspotPlayerId }}',
-      env: {
-        "gatewayServerEndpoint": '{{ site.gateway_server_endpoint }}'
-      }
-    };
-  </script>
-  {% javascript_link_tag application %}
-  {% javascript_link_tag application_deferred async %}
-  {% if page.masonry_js %}
-  {% javascript_link_tag masonry_deferred async %}
-  {% endif %}
-
-  {% if page.title == "Atrium Events" %}
-    {% javascript_link_tag events async %}
-  {% endif %}
-</body>
+  <body class="crds-styles">
+    {% include _gtm-body.html %} {% if page.requires_auth %} {% include
+    _preloader.html %} {% endif %}
+    {{ content }}
+    {% include _monetate-body.html %}
+    <script>
+      var CRDS = {
+        media: {
+          cms: "{{ site.shared_header.cms }}",
+          app: "{{ site.shared_header.app }}",
+          img: "{{ site.shared_header.img }}",
+          prefix: "{{ site.shared_header.prefix }}"
+        },
+        streamspotId: "{{ site.streamspotId }}",
+        streamspotKey: "{{ site.streamspotKey }}",
+        streamspotPlayerId: "{{ site.streamspotPlayerId }}",
+        env: {
+          gatewayServerEndpoint: "{{ site.gateway_server_endpoint }}"
+        }
+      };
+    </script>
+    {% javascript_link_tag application %} {% javascript_link_tag
+    application_deferred async %} {% if page.masonry_js %} {%
+    javascript_link_tag masonry_deferred async %} {% endif %} {% if page.title
+    == "Atrium Events" %} {% javascript_link_tag events async %} {% endif %}
 
+    <!-- we are crossroads -->
```

Comment:
could we put this on the actual index.html?

Just for clarity, we decided as a team that / will NOT be coming to contentful. So instead of every page having that, only the one that decides to be blank will have it.